### PR TITLE
ci: DOMA-3818 added new lint rules

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -1,35 +1,62 @@
...
+            2,
             'always',
-            ({ header }) => {
-                const standalonePrefixes = ['HOTFIX', 'INFRA', 'REFACTOR']
-                for (let prefix of standalonePrefixes) {
-                    if (header.match(new RegExp(`^${prefix} \\w+`, 'i'))) {
-                        return [true]
-                    }
+            ({ subject }) => {
+                if (!subject || !subject.match(/DOMA-\d+(\s+\S+){2,}/)) {
```
Comment (sitozzz) [inline commitlint.config.js:39]: What about non task prefixes? Like INFRA, HOTFIX and etc?

File: commitlint.config.js [hunk 2]
```diff
@@ -1,35 +1,62 @@
...
+            },
+        ],
+        'function-rules/subject-empty': [
+            2,
+            'always',
+            ({ subject }) => {
+                if (!subject || !subject.match(/^DOMA-\d+/)) {
+                    return [
+                        false,
+                        'Wrong commit subject. Commit subject must starts with task number. Allowed formats: DOMA-123',
```
Comment (akarjakin) [inline commitlint.config.js:56]: `must starts` -> `should start`

File: commitlint.config.js [hunk 3]
```diff
@@ -1,35 +1,62 @@
...
-                    return [false, `Commit message is too short. ${detailsPrompt}`]
+
+                return [true]
+            },
+        ],
+        'function-rules/subject-empty': [
+            2,
+            'always',
+            ({ subject }) => {
+                if (!subject || !subject.match(/^DOMA-\d+/)) {
```
Comment (akarjakin) [inline commitlint.config.js:53]: better to move static regexps to constants

File: commitlint.config.js (hunk 1 above)
Comment (akarjakin) [inline commitlint.config.js:39]: better to move static regexps to constants

**Review by akarjakin — APPROVED** [review]

File: commitlint.config.js (hunk 1 above)
Comment (SavelevMatthew) [reply commitlint.config.js]: Added INFRA subject prefix, added hot fix type which does not require a task number

**Review by sitozzz — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

**Pull request merged**